### PR TITLE
Update upgrade-test-harness following 1.8.0 release

### PR DESCRIPTION
**Pull request by sebgl**

### CODE REVIEW

File: hack/upgrade-test-harness/conf.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ testParams:
...
     operatorVersion: 1.7.1
     stackVersion: 7.14.0
-  - name: upcoming
+  - name: v180
     operatorVersion: 1.8.0
-    stackVersion: 7.14.0
+    stackVersion: 7.15.0
+  - name: upcoming
+    operatorVersion: 1.9.0
+    stackVersion: 7.15.0
```
Comment (thbkrkr) [inline hack/upgrade-test-harness/conf.yaml:43]:
```suggestion
    stackVersion: 7.16.0
```

Comment (sebgl) [reply hack/upgrade-test-harness/conf.yaml:43]: that version does not exist yet so I guess the test would just fail?

Comment (thbkrkr) [reply hack/upgrade-test-harness/conf.yaml:43]: @#! I meant `7.16.0-SNAPSHOT`.

Comment (sebgl) [inline hack/upgrade-test-harness/conf.yaml:43]:
```suggestion
    stackVersion: 7.16.0-SNAPSHOT
```

**Review by thbkrkr — APPROVED** [review]

**Pull request merged**